### PR TITLE
videointelligence: add quickstart

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

File: videointelligence/video_quickstart/main.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		log.Fatalf("Failed to start annotation job: %v", err)
+	}
+
+	resp, err := op.Wait(ctx)
+	if err != nil {
+		log.Fatalf("Failed to annotate: %v", err)
+	}
+
+	// Only one video was processed, so get the first result.
+	result := resp.GetAnnotationResults()[0]
```
Comment (odeke-em) [inline videointelligence/video_quickstart/main.go]: How about:
```
annotations := resp.GetAnnoationResults()
if len(annotations) == 0 {
   log.Fatalf("No annotations found")
}
// Only one video was processed, so get the first
results := annotations[0]
```

Comment (broady) [reply videointelligence/video_quickstart/main.go]: I think this is guaranteed to be len==1.

**Comment by broady** [comment]
R= @remi @tswast

File: videointelligence/video_quickstart/main.go (hunk 1 above)
Comment (odeke-em) [reply videointelligence/video_quickstart/main.go]: Oh okay, gotcha, thanks for that update. In that case, you've got my vote Mr. President!

Comment (broady) [reply videointelligence/video_quickstart/main.go]: I'll wait for @remi (Cloud ML API expert) to confirm.

File: videointelligence/video_quickstart/main.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	ctx := context.Background()
+
+	// Creates a client.
+	client, err := video.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
+
+	op, err := client.AnnotateVideo(ctx, &videopb.AnnotateVideoRequest{
+		InputUri: "gs://demomaker/cat.mp4",
```
Comment (tswast) [inline videointelligence/video_quickstart/main.go]: I created a `cloud-storage-samples` bucket that might make sense to use for samples like this.

Comment (broady) [reply videointelligence/video_quickstart/main.go]:
```
AccessDeniedException: 403 cbro@google.com does not have storage.objects.list access to cloud-storage-samples.
```

Comment (tswast) [reply videointelligence/video_quickstart/main.go]: Oops, I forgot to make it public. I've made you owner of the `storage-samples` project that owns that bucket. I'd make it public myself but am WFH without a corp cert right now.

Comment (tswast) [reply videointelligence/video_quickstart/main.go]: Ah, I can't make you an owner with the app, either. @frankyn might be able to help.

Comment (frankyn) [reply videointelligence/video_quickstart/main.go]: You are an admin now @broady!

Comment (broady) [reply videointelligence/video_quickstart/main.go]: Merging now without this change. Please file a bug if this URI isn't appropriate for the quickstart.

**Pull request merged**